lms/djangoapps/lti_provider/users.py

```diff
@@ -15,8 +15,12 @@
 from lti_provider.models import LtiUser
 from student.models import UserProfile
 
+USERNAME_SIZE = 30 #size of the field in the database
+FIRST_NAME_SIZE = 30 #size of the field in the database
+LAST_NAME_SIZE = 30 #size of the field in the database
+EMAIL_SIZE = 75 #size of the field in the database
 
-def authenticate_lti_user(request, lti_user_id, lti_consumer):
+def authenticate_lti_user(request, lti_user_id, lti_consumer, lti_email=None, lti_user_first_name=None, lti_user_last_name=None):
```

Comment:
Seems like these optional demographics could become pretty lengthy, perhaps a dict would be better than a list of args?